lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -52,6 +52,7 @@ def call(params)
           domain_name = params.organization || params.domain
           domain = client.domain(domain_name) if domain_name
           project = client.projects(params.gdc_project) || client.projects(params.gdc_project_id)
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)